core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly

```diff
@@ -30,88 +30,121 @@ THE SOFTWARE.
   <l:layout permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%System Information}">
     <st:include page="sidepanel.jelly" />
     <l:main-panel>
-        <l:hasPermission permission="${app.SYSTEM_READ}">
+      <l:hasPermission permission="${app.SYSTEM_READ}">
+        <div class="jenkins-app-bar">
+          <div class="jenkins-app-bar__content">
             <h1>${%System Properties}</h1>
```

Comment:
Theres a little bit of a spacing issue with the headings on this page but not a blocker imo, will be easy to resolve once the spacing classes are merged in from https://github.com/jenkinsci/jenkins/pull/5916#issuecomment-968352585